streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -406,24 +361,6 @@ private void cleanGlobal(final String intermediateUserTopic) {
         Assert.assertEquals(0, exitCode);
     }
 
-    private void waitUntilUserTopicsAreDeleted() {
```

Comment:
`userTopicExists` is actually an unused method now -- we can delete it.

However, we could also add more functions to `EmbeddedKafkaCluster` to check a topic exists or not. Not sure how common this is though.